packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -575,25 +703,37 @@ export class MatchingDataAttestationGroup {
         continue;
       }
 
-      let notSeenAttesterCount = 0;
+      if (excluded.has(attestation)) {
```

Comment:
Nit: Checking if an `Attestation` exists in a set seems weird. Would prefer something "lighter" like root hex, or at the very least, `AttestationData` when comparing.